### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by danforbes**
This pull request assumes the existence of a `rococo-registration` label.

### CODE REVIEW

**Comment by danforbes** [comment]
You can see these "in action" on my fork https://github.com/danforbes/cumulus/issues/new/choose

File: .github/ISSUE_TEMPLATE/register-rococo-parachain.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+assignees: ''
+
+---
+
+# About Your Parachain
+
+- Name: 
+- ID: 
+- Purpose: 
+- GitHub:
```
Comment (bkchr) [inline .github/ISSUE_TEMPLATE/register-rococo-parachain.md:15]: I think we need a riot handle as well IMHO.

Comment (danforbes) [reply .github/ISSUE_TEMPLATE/register-rococo-parachain.md:15]: See below :wink:

File: .github/ISSUE_TEMPLATE/register-rococo-parachain.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ assignees: ''
...
 
+# Rococo Validator
+
+_In order to register a Rococo parachain, you will need to maintain a validator for the Rococo relay chain._
+
+- Validator address: 
+
 # Contact Information
 
 - [Riot](https://app.element.io/#/room/!WuksvCDImqYSxvNmua:matrix.parity.io?via=matrix.parity.io&via=matrix.org&via=web3.foundation) username:
```
Comment (JoshOrndorff) [inline .github/ISSUE_TEMPLATE/register-rococo-parachain.md:25]:
```suggestion
- [Matrix](https://app.element.io/#/room/!WuksvCDImqYSxvNmua:matrix.parity.io?via=matrix.parity.io&via=matrix.org&via=web3.foundation) username:
```

**Pull request closed without merging**